main.go

```diff
@@ -31,8 +33,12 @@ const (
 // tenantRulesLoader represents logic for loading and filtering PrometheusRule objects by tenants.
 // Useful for testing without spinning up cluster.
 type tenantRulesLoader interface {
+	getLokiAlertingRules() ([]*lokiv1beta1.AlertingRule, error)
+	getLokiRecordingRules() ([]*lokiv1beta1.RecordingRule, error)
 	getPrometheusRules() ([]*monitoringv1.PrometheusRule, error)
-	getTenantRuleGroups(prometheusRules []*monitoringv1.PrometheusRule) map[string]monitoringv1.PrometheusRuleSpec
+	getTenantLokiAlertingRuleGroups(alertingRules []*lokiv1beta1.AlertingRule) map[string]lokiv1beta1.AlertingRuleSpec
```

Comment:
Maybe this should be same naming pattern as metrics?
```suggestion
	getTenantLogsAlertingRuleGroups(alertingRules []*lokiv1beta1.AlertingRule) map[string]lokiv1beta1.AlertingRuleSpec
```